tests/baselines/reference/restTupleElements1.errors.txt

```diff
@@ -94,14 +92,12 @@ tests/cases/conformance/types/tuple/restTupleElements1.ts(59,4): error TS2345: A
                                   ~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2344: Type '[number, number, string]' does not satisfy the constraint '[number, ...number[]]'.
 !!! error TS2344:   Type at positions 1 through 2 in source is not compatible with type at position 1 in target.
-!!! error TS2344:     Type 'string | number' is not assignable to type 'number'.
-!!! error TS2344:       Type 'string' is not assignable to type 'number'.
+!!! error TS2344:     Type 'string' is not assignable to type 'number'.

```

Comment:
I'm ok with that change. The message could be better, because what ultimately fails is the type at position 2 in source, so we could be more specific. But considering we had the same problem before this PR, it seems out of scope for this PR.

Reply:
I could improve on this error message here or in a followup PR - I would need a confirmation on how this should be reported though. I guess that in this case, I should drop the "positions X through Y" bit and just report this error for the fixed~ position. IIRC this would mean that `TS2344` would no longer be reported anywhere - and it's the primary reason why I've left it like this. I wasn't sure how I should approach removing an existing error code.